packages/vuetify/src/components/VSelect/__tests__/VSelect.spec.cy.tsx

```diff
@@ -263,4 +263,56 @@ describe('VSelect', () => {
       cy.get('.v-overlay__content .v-list-item .v-list-item-title').eq(1).should('have.text', 'Item 4')
     })
   })
+
+  describe('global configuration', () => {
```

Comment:
We should pay more attention when dealing with nested components. `VSelect` has three levels of nested components. Do you think this test case covers everything in this senario? @johnleider @KaelWD

Reply:
I'm okay with the test. I suspect inputs are the only components we would have this issue with due to their complex nature of applying properties/attributes.

Reply:
VSelect nested structure

```ts
<VSelect>
   <VTextField>
      <VInput>
```